src/pages/Home.vue

```diff
@@ -71,6 +78,8 @@ export default {
   data() {
     return {
       groupData: null,
+      toggle: true,
```

Comment:
You shouldn't be using a variable to display the language picker. Because this way, everytime the app is refreshed, the picker shows up. It should show up only when the user is coming on to the page for the first time. Refer to how it is done in Plio.

Reply:
okay